app/app.R

```diff
@@ -20,23 +20,54 @@ ui <- navbarPage(
     windowTitle = apptitle,
     theme = "styling.css",
     position="fixed-top", collapsible = TRUE,
-    tabPanel("Results", sidebarLayout(position = "right",
+    tabPanel("Results", sidebarLayout(position = "left",
       sidebarPanel(
-        numericInput("R",
-          "R:",
-          min = 0.5,
-          max = 10,
-          value = 2
+        dateInput("admissionDate",
+          "Date of admission:",
+          format = "dd/mm/yyyy"
         ),
-        numericInput("SI",
-                     "serial interval:",
-                     min = 1,
+        numericInput("numberAdmissions",
+                     "Number of admissions on that date:",
+                     min = 0,
+                     max = 10000,
+                     value = 1
+        ),
+        numericInput("assumedReporting",
+                     "Reporting rate (%):",
+                     min = 0,
+                     max = 100,
+                     value = 100
+        ),
+        numericInput("doublingTime",
+                     "Assumed doubling time (days):",
+                     min = 0,
+                     max = 10,
+                     value = 2
+        ),
+        numericInput("uncertaintyDouiblingTime",
+                     "Uncertainty in doubling time (days):",
+                     min = 0,
                      max = 10,
-                     value = 5
+                     value = 2
+        ),
+        radioButtons(inputId="distributionDuration", label="Distribution of duration of stay", 
+                     choices=c("non-critical hospitalization","critical hospitalization","user-defined")
```

Comment:
My bad for putting it in originally, but could you:

1. remove the 'user-defined' choice
2. add an issue for implementing user-defined duration of hospitalisation

I don't want to rush this for now, it'd be good to discuss how we implement this with the group. Also this should trigger another GUI input (reactive UI) which is a tiny added complexity.

Reply:
Yeah - I just didn't know what that input would look like so left it off. But have removed the user choice for now.